src/main/java/org/springframework/cloud/deployer/spi/kubernetes/AbstractKubernetesDeployer.java

```diff
@@ -308,6 +312,26 @@ protected ImagePullPolicy deduceImagePullPolicy(AppDeploymentRequest request) {
 		return requests;
 	}
 
+	/**
+	 * Get the hostNetwork setting for the deployment request.
+	 *
+	 * @param request The deployment request.
+	 * @return Whether host networking is requested
+	 */
+	protected boolean getHostNetwork(AppDeploymentRequest request) {
+		String hostNetworkOverride =
+				request.getDeploymentProperties().get("spring.cloud.deployer.kubernetes.hostNetwork");
+		boolean hostNetwork;
```

Comment:
I'm OK with either way - we do follow the pattern I used for other properties, but I'm OK with your way as well.